crates/hotshot/task-impls/src/quorum_vote/handlers.rs

```diff
@@ -219,6 +219,12 @@ pub(crate) async fn handle_quorum_proposal_validated<
                 leaf_info.leaf.view_number(),
                 leaf_info.leaf.block_header().block_number(),
             );
+
```

Comment:
Rather than do this here, can we do it in the same place we broadcast the external decide event?